<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMDataViewActionsBar</class>
 <widget class="QFrame" name="AMDataViewActionsBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="openSeparateEditorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Open in Separate Editors</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	border-top: 1px solid transparent;
}

QToolButton:pressed {
border-top: 3px solid transparent;
}

QToolButton {
border: none;
}

</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/32x32/edit-find-replace.png</normaloff>:/32x32/edit-find-replace.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openSameEditorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Open in Same Editor</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	border-top: 1px solid transparent;
}

QToolButton:pressed {
border-top: 3px solid transparent;
}

QToolButton {
border: none;
}

</string>
       </property>
       <property name="text">
        <string>Compare</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/32x32/preferences-desktop-theme.png</normaloff>:/32x32/preferences-desktop-theme.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exportButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	border-top: 1px solid transparent;
}

QToolButton:pressed {
border-top: 3px solid transparent;
}

QToolButton {
border: none;
}

</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/32x32/system-file-manager.png</normaloff>:/32x32/system-file-manager.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="scanConfigurationButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Show the configuration for the selected scan(s).</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	border-top: 1px solid transparent;
}

QToolButton:pressed {
border-top: 3px solid transparent;
}

QToolButton {
border: none;
}

</string>
       </property>
       <property name="text">
        <string>Config</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/32x32/hammer-wrench.png</normaloff>:/32x32/hammer-wrench.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>126</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="expandAllButton">
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	border: 1px solid transparent;
}

QToolButton:pressed {
	border: 1px solid gray;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(233, 233, 233, 255), stop:0.502513 rgba(199, 198, 198, 255), stop:1 rgba(163, 162, 162, 255));
}

QToolButton {
border: none;
}

</string>
       </property>
       <property name="text">
        <string>Expand All</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/32x32/list-add.png</normaloff>:/32x32/list-add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="collapseAllButton">
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	border: 1px solid transparent;
}

QToolButton:pressed {
	border: 1px solid gray;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(233, 233, 233, 255), stop:0.502513 rgba(199, 198, 198, 255), stop:1 rgba(163, 162, 162, 255));
}

QToolButton {
border: none;
}

</string>
       </property>
       <property name="text">
        <string>Collapse All</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/icons.qrc">
         <normaloff>:/32x32/list-remove.png</normaloff>:/32x32/list-remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sizeIconSmall">
     <property name="maximumSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../../icons/icons.qrc">:/32x32/folder.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="sizeSlider">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sizeIconLarge">
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../../icons/icons.qrc">:/32x32/folder.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
